docs/usage.md

```diff
@@ -0,0 +1,167 @@
+
+## Job Resources
+### Automatic resubmission
+Each step in the pipeline has a default set of requirements for number of CPUs,
+memory and time. For most of the steps in the pipeline, if the job exits
+on UPPMAX with an error code of `143` (exceeded requested resources) it will
+automatically resubmit with higher requests (2*original, then 3*original).
+If it still fails after three times then the pipeline is stopped.
+
+### Custom resource requests
+Wherever process-specific requirements are set in the pipeline, the default
+value can be overwritten with config variables. These can be set on the command
+line or in a config file. The names are set as `[process name]_[resource type]`,
+for example `star_memory`.
+
+So, to override the defaults for STAR, you can do run the pipeline as follows:
+```bash
+nextflow run SciLifeLab/NGI-RNAseq --star_cpus 1 --star_memory '10 GB' --star_time '24h'
+```
+
+Alternative, these can be set in a config file:
+```groovy
+params {
+  star_cpus = 1
+  star_memory = '10 GB'
+  star_time = '24h'
+}
+```
+
+## Other command line parameters
+### `--outdir`
+The output directory where the results will be saved.
+
+### `--sampleLevel`
+Used to turn of the edgeR MDS and heatmap. Set automatically when running on fewer than 3 samples.
+
+### `--strandRule`
+Some RSeQC jobs need to know the stranded nature of the library. By default, the pipeline will use
+`++,--` for single end libraries and `1+-,1-+,2++,2--` for paired end libraries. These codes are for
+strand specific libraries (antisense). `1+-,1-+,2++,2--` decodes as:
+
+*  Reads 1 mapped to `+` => parental gene on `+`
+*  Reads 1 mapped to `-` => parental gene on `-`
+*  Reads 2 mapped to `+` => parental gene on `-`
+*  Reads 2 mapped to `-` => parental gene on `+`
+
+Use this parameter to override these defaults. For example, if your data is paired end and strand specific,
+but same-sense to the reference, you could run:
+```
+nextflow run NGI-RNAseq/main.nf --strandRule '1++,1--,2+-,2-+'
+```
+Use `--strandRule 'none'` if your data is not strand specific.
+
+### `--rlocation`
+Some steps in the pipeline run R with required modules. By default, the pipeline will install
+these modules to `~/R/nxtflow_libs/` if not present. You can specify what path to use with this
+command line flag.
+
+### `--clusterOptions`
+Submit arbitrary SLURM options (UPPMAX profile only). For instance, you could use `--clusterOptions '-p devcore'`
+to run on the development node (though won't work with default process time requests).
+
+### `-c`
+Specify the path to a specific config file (this is a core NextFlow command). Useful if using different UPPMAX
+projects or different sets of reference genomes. **NB:** one hyphen only (core Nextflow parameter).
+
+## Stand-alone scripts
+There is a folder with some unmaintained standalone scripts that you can use:
+[stand-alone scripts](https://github.com/SciLifeLab/NGI-RNAseq/blob/master/stand-alone-scripts).
```

Comment:
This link is broken and the paragraph is outdated. Maybe we should mention the bin directory were the executed scripts are stored and that there are standalone stuff in there?